AdvancedPlaybackSampleApp/AdvancedPlaybackSampleApp/Lists/AdvancedPlaybackListViewController.m

```diff
@@ -48,8 +50,9 @@ - (void)insertNewObject:(PlayerSelectionOption *)selectionObject {
   if (!self.options) {
       self.options = [[NSMutableArray alloc] init];
   }
-  [self.options insertObject:selectionObject atIndex:0];
```

Comment:
I did this because the list was ending up reversed when shown on-screen; this makes them appear in the same order they are aded via insertNewObject (above).